mk/cfg/s390x-unknown-linux-gnu.mk

```diff
@@ -1 +1,24 @@
-# rustbuild-only target
+# s390x-unknown-linux-gnu configuration
+CROSS_PREFIX_s390x-unknown-linux-gnu=s390x-linux-gnu-
+CC_s390x-unknown-linux-gnu=$(CC)
+CXX_s390x-unknown-linux-gnu=$(CXX)
+CPP_s390x-unknown-linux-gnu=$(CPP)
+AR_s390x-unknown-linux-gnu=$(AR)
+CFG_LIB_NAME_s390x-unknown-linux-gnu=lib$(1).so
+CFG_STATIC_LIB_NAME_s390x-unknown-linux-gnu=lib$(1).a
+CFG_LIB_GLOB_s390x-unknown-linux-gnu=lib$(1)-*.so
+CFG_LIB_DSYM_GLOB_s390x-unknown-linux-gnu=lib$(1)-*.dylib.dSYM
+CFG_CFLAGS_s390x-unknown-linux-gnu := -m64 $(CFLAGS)
```

Comment:
Well, the last few generations of System Z processors are z10, z196, zEC12, and z13.  In this order, each more recent processor is 100% upwards compatible with all earlier ones.  Or to put it the other way, you can link together objects compiled for any of the processors, as long as you run the resulting binary on the most recent one.  Therefore, I don't think it is particularly necessary to force the C components of std to be built for z10.

(The exception might be if you also run into problems with the valgrind tests since valgrind doesn't yet support all zEC12 instructions.  That might indeed be a reason to use an older march level, until valgrind is fixed ...)